crates/plugins/mediator-coordination/src/model/stateful/entity.rs

```diff
@@ -22,6 +23,14 @@ pub struct Connection {
     /// Generated DID to route messages to client.
     pub routing_did: String,
 }
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct Messages {
+    #[serde(rename = "_id")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub id: Option<ObjectId>,
+    pub message: Message,
+    pub recipient_did: String,
+}
```

Comment:
Prefer a singular name.
```suggestion
#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
pub struct RoutedMessage {
    #[serde(rename = "_id")]
    #[serde(skip_serializing_if = "Option::is_none")]
    pub id: Option<ObjectId>,
    pub message: Message,
    pub recipient_did: String,
}
```

Reply:
okay i guess its more descriptive